protocols/kad/src/query/peers/closest/disjoint.rs

```diff
@@ -773,11 +774,11 @@ mod tests {
 
             // Make each peer aware of a random set of other peers within the graph.
             for (peer_id, peer) in peers.iter_mut() {
-                peer_ids.shuffle(g);
+                peer_ids.shuffle(&mut rand::thread_rng());
```

Comment:
This makes the `Arbitrary` implementation on `Graph` no longer deterministic based on `Gen`, correct? Can we stay deterministic?

Reply:
Still there is a `random_peers` with rand. So maybe there is a way to stay deterministic, but I have no idea how to achieve this in an easy way.

Reply:
`random_peers` just needs a `[u8; 32]`, right? That can be generated with `Gen`, right?